### ISSUE TITLE
Deprecation rule should surface the deprecation message

### ISSUE THREAD

**Issue by kenyee** [detekt/detekt#6598]
## Expected Behavior
When a class has a deprecation annotation like this:
```
@Deprecation("some reason")
class Lookup
```

I expect Detekt's PossibleBugs/Deprecation rule to surface that message to users in the lint warning.

## Current Behavior
The error message is just:
```
Deprecation - [Lookup is deprecated.]
```

## Context
I'm trying to surface deprecated classes/methods to users along with the reasons they're deprecated.

**Comment by BraisGabin** [comment]
I implemented #6614 that should fix this. Could you check if that would help for your use case? (Help with the wording of the message is more than welcome)

**Comment by kenyee** [comment]
Looks good.  Only suggestion is maybe to use "because" instead of "with message" since the deprecation message is the reason usually.
